plugins/by-name/kiwi/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "kiwi";
+  package = "kiwi-nvim";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  settingsExample = {
+    __unkeyed-1 = {
+      name = "work";
+      path = "work-wiki";
+    };
+    __unkeyed-2 = {
+      name = "personal";
+      path = "personal-wiki";
+    };
+  };
```

Comment:
It'd be great for plugins like this one to be able to drop the submodule type entirely. It is unnecessary when we aren't declaring sub-options and it prevents us from using a more inclusive type for `settings`.

In this case we probably want `either listOf attrsOf`.